styles.css

```diff
@@ -34,34 +34,60 @@ main {
 }
 
 section {
-  border: solid 2px darkred;
+  border: solid 4px darkred;
   display: flex;
   flex-direction: column;
   width: 90%;
   margin: auto;
   padding: 20px 20px;
-  align-items: center;
   list-style: none;
 }
 
 ul {
   display: flex;
   flex-direction: row;
-  justify-items: flex-end;
+  justify-items: end;
+  align-items: flex-start;
   list-style: none;
+  padding-top: 15px;
 }
 
 .flexbox-container {
+  position: relative;
+  top: -25px;
+  right: auto;
+  /*nach dem einfügen der oberen 3 zeilen,
+flexbox-container über dem header beim
+scrollen; bookmark ist nicht rechts positioniert.*/
   background-color: rgb(119, 195, 198);
+  text-align: center;
+  color: rgb(5, 5, 107);
+}
+
+.bookmark {
+  width: 45px;
+  position: absolute;
+  top: -25px;
```

Comment:
I think you need to add: `right: 0;` to this CSS rule as this handles the positioning of your bookmark.

Reply:
Vielen Danke für die ganzen Tipps!! 😊